### ISSUE TITLE
Support changing language for properties search

### ISSUE THREAD

**Issue by dpriskorn** [derenrich/wwwyzzerdd#7]
@salgo60 would like to use Swedish property names :)

**Comment by derenrich** [comment]
yeah this is on my todo list. not 100% sure how i'm going to do it. maybe just use the language of the wiki as the language of the properties as a first pass.

**Comment by salgo60** [comment]
nice and have a fallback language I guess... the rate WD gets new properties is faster than the people translating the names is my feeling ;-)
